src/main.rs

```diff
@@ -104,8 +104,21 @@ async fn handle_message(bot: Bot, msg: Message) {
                         req.parse_mode = ParseMode::Markdown;
                         req.disable_web_page_preview = Some(true);
 
+                        bot.send(&req).await.unwrap();
+                    } else {
+                        let mut req =
+                            SendMessage::new(msg.chat.id, messages::NEED_REPLY_TEXT_MESSAGE);
+                        req.parse_mode = ParseMode::Markdown;
+                        req.disable_web_page_preview = Some(true);
```

Comment:
These two lines don't need to be here since the message we're sending is just plaintext without any links.

```suggestion
```